ios/Localization.md

```diff
@@ -0,0 +1,32 @@
+# Localization
+
+We take two different approaches when localizing iOS apps. Primarily, we use POEditor with Fastlane plugin
+to synchronize it by calling one command. Alternative approach is to use ACKLocalization tool synchronized
+with a Google sheet. Optionally, we use R.swift to check that localization resources are synchronized at compile time.
```

Comment:
```suggestion
with a Google sheets. Optionally, we use R.swift to check that localization resources are synchronized at compile time.
```

Reply:
"a" + množné číslo se mi nezdá.
Dal bych with "Google Sheets document" a nebo nechal "a Google sheet".

Reply:
Je to tak jak říká Adam.

Reply:
Jj v pohode to je drobnost. Ja som to bral ako "nazov" nie ako mnozne cislo.. Treba i na googlu to pisu ako _Google Sheets **is** a spreadsheet..._